src/plugins/intel_cpu/src/nodes/fake_quantize.cpp

```diff
@@ -1896,6 +1896,7 @@ static float roundHalfToEven(float f) {
 }
 
 void FakeQuantize::updateOptimizedFormula(bool do_rounding) {
+    static const float simInfinity = std::pow(2, 100);
```

Comment:
Just Like the ONNX model, I think the special FQ in IR models also should be supported in LPT transformation which affects less in execution time.

Reply:
Thanks a lot, that's good idea, the fix is changed accordingly!